defaults/main.yml

```diff
@@ -28,7 +28,6 @@ mssql_rpm_key: https://packages.microsoft.com/keys/microsoft.asc
 mssql_server_repository: "{{ __mssql_server_repository }}"
 mssql_client_repository: "{{ __mssql_client_repository }}"
 mssql_ha_configure: false
-mssql_ha_firewall_configure: false
```

Comment:
don't you need `mssql_manage_firewall: false` here?

Reply:
Thanks, fixed this.

Reply:
oops - typo